src/libs/renderer/src/sdevice.cpp

```diff
@@ -1669,6 +1676,30 @@ bool DX9RENDER::TextureLoad(long t)
     return true;
 }
 
+bool DX9RENDER::TextureLoadUsingD3DX(const char* path, long t)
+{
+    // TODO: reimplement the whole thing in a tidy way
+    IDirect3DTexture9 *pTex;
+    if(CHECKD3DERR(D3DXCreateTextureFromFileA(d3d9, path, &pTex)))
```

Comment:
shouldn't we call the Unicode function and convert our utf8 path to utf16?

Reply:
but I guess if we leave it as it is, textures won't be loaded if there are non-ascii characters in the texture path, right?

Reply:
In this case, no, the path is relative to %CD%. And since there is no Unicode support for most asset paths I highly doubt it may cause any issues with the existing games :)

Reply:
Most issues start when using %USERPROFILE%. I.e. for logging, savedata, sentry-db, etc.
Everything that lays in the %CD% subfolders normally should not use absolute paths at all. And therefore should not cause issues with Unicode path. At least until we want to introduce it in the subfolder structure... do we?

Reply:
personally, I'd prefer to use absolute paths for everything.
okay, in this case it's fine to leave it as it is for now. thanks for the clarification!